sites/main-site/src/content/special-interest-groups/meta-omics/index.md

```diff
@@ -30,7 +30,9 @@ The initial aims of this interest group will be:
 1. Helping coordinate compatibility between existing and potential meta-omic nf-core pipelines
 2. Implementing semi-'chaining' of sequential pipelines
 3. Developing standard parameters for different sub-disciplines (i.e. microbiome analysis) for each pipeline
-4. Documenting usage of 'first level' meta-omic pipelines followed by complementary analyses with "secondary" pipelines such as #funcscan, #phageannotator, #phyloplace, #rnaseq, #differentialabundance etc.
+4. Documenting usage of 'first level' meta-omic pipelines (createtaxdb, ampliseq, eager, taxprofiler, mag, metatdenovo) followed by complementary analyses with "secondary" pipelines such as (funcscan, phageannotator, phyloplace, rnaseq, differentialabundance) etc.
+
+![London Underground style 'tube map' showing how potentially nf-core pipelines can be used for complementary and sequential analysis within meta-omics disciplines](nf-core-meta-omics-metro.png)
```

Comment:
I think this version might stress better that the important bit is what is showing and not the metro map, which might be 😁 
```suggestion
![Showcasing how nf-core pipelines can potentially be used for complementary and sequential analysis within meta-omics disciplines using a London Underground style 'tube map'](nf-core-meta-omics-metro.png)
```

Reply:
Updated!